### PR TITLE
TNO-69 Add List View

### PR DESCRIPTION
**Pull request by Fosol**
@jdtoombs The List View feature is not complete, however this PR is to provide shared functionality between our work.  I have publish tno-core v0.0.85 with updates to most of the dependencies.  I have created common components for forms.  Currently they reside in the main package.  We'll move them to `tno-core` once everything is working.

Going forward we'll merge into the `app` branch until at least one of us has completed our work and it's ready to merge with `dev`.

### CODE REVIEW

File: app/editor/package.json [hunk 1]
```diff
@@ -6,49 +6,50 @@
   "private": false,
```
Comment (Fosol) [inline app/editor/package.json:6]: Upgraded as many dependencies as I could without breaking it.

File: app/editor/src/components/router/PrivateRoute.tsx [hunk 2]
```diff
@@ -41,12 +41,10 @@ export const PrivateRoute = ({
...
 
-  if (
-    !keycloak.authenticated ||
-    (!!claims && !keycloak.hasClaim(claims)) ||
-    (!!roles && !keycloak.hasRole(roles))
-  ) {
+  if (!keycloak.authenticated) {
     return <Navigate to={redirectTo} />;
+  } else if ((!!claims && !keycloak.hasClaim(claims)) || (!!roles && !keycloak.hasRole(roles))) {
+    return <div>ERROR</div>;
```
Comment (Fosol) [inline app/editor/src/components/router/PrivateRoute.tsx:47]: This needs to be written to load the appropriate error page.

File: app/editor/src/features/login/Login.tsx [hunk 3]
```diff
@@ -9,10 +8,9 @@ import { useKeycloakWrapper } from '../../hooks';
  */
 export const Login = () => {
   const keycloak = useKeycloakWrapper();
-  const navigate = useNavigate();
 
   if (keycloak.authenticated) {
-    navigate('/');
+    return <Navigate to="/" />;
```
Comment (Fosol) [inline app/editor/src/features/login/Login.tsx:13]: Fixed bug with latest version of react-router.

File: libs/npm/core/.devcontainer/devcontainer.json [hunk 4]
```diff
@@ -22,7 +22,7 @@
     "esbenp.prettier-vscode",
     "rvest.vs-code-prettier-eslint",
     "formulahendry.auto-close-tag",
-    "cssho.vscode-svgviewer",
+    "jock.svg",
```
Comment (Fosol) [inline libs/npm/core/.devcontainer/devcontainer.json:25]: A better viewer

File: libs/npm/core/package.json [hunk 5]
```diff
@@ -31,7 +31,6 @@
     "react-error-overlay": "6.0.9",
```
Comment (Fosol) [inline libs/npm/core/package.json:31]: I'd love to remove all the dependencies we don't need in this shared library.

File: libs/npm/core/src/hooks/keycloak/useKeycloakEventHandler.ts [hunk 6]
```diff
@@ -28,7 +28,7 @@ export const useKeycloakEventHandler = () => {
           state.setAuthReady(true);
         } else {
           //TODO: log error properly
-          console.debug(`keycloak event: ${eventType} error ${error}`);
+          console.debug(`keycloak event: ${eventType} error`, error);
```
Comment (Fosol) [inline libs/npm/core/src/hooks/keycloak/useKeycloakEventHandler.ts:31]: Hopefully it'll include the error information in the output.  This however shouldn't be a console.debug statement.

**Review by jdtoombs — APPROVED** [review]
Looks good, thanks for this! Will switch in some of these into my current work

File: app/editor/src/components/form/dropdown/Dropdown.tsx [hunk 7]
```diff
@@ -0,0 +1,77 @@
+import React, { SelectHTMLAttributes } from 'react';
+
+import { DropdownVariant, instanceOfIOption, IOption } from '.';
+import * as styled from './DropdownStyled';
+
+export interface IDropdownProps extends SelectHTMLAttributes<HTMLSelectElement> {
+  /**
+   * The control name.
+   */
+  name: string;
```
Comment (Fosol) [inline app/editor/src/components/form/dropdown/Dropdown.tsx:10]: Moving `name` and `label` from the formik version to the base version.  Need to do this for the other components too.

**Pull request merged**